### PR TITLE
Support for Kube users

### PR DESCRIPTION
**Pull request by benarent**
Opening another draft PR since I accidentally merged the older one into 4.3 base.

### CODE REVIEW

**Comment by benarent** [comment]
This PR has already been reviewed, but I merged into the base too early - Going to keep all 4.3 docs as drafts, as we are about a month out, and other base changes are happening to the docs.

File: docs/4.3/enterprise/ssh_sso.md [hunk 1]
```diff
@@ -121,12 +121,18 @@ spec:
...
-have a continuation of identity, but often the first part of the email. The local part. 
-Is useful to send to other systems as it's unique.  Teleport 4.3 has extended the variable
-interpolation, to extract this local part. 
-
+Along with sending groups, an SSO provider will also provide a user's email address. 
+In many organizations, the username that a person uses to log into a system is the 
+same as the first part of their email address - the 'local' part. For example, `dave.smith@acme.com`
+ might log in with the username `dave.smith`. Teleport 4.2.6+ adds an easy way to 
+ extract the first part of an email address so it can be used as a username - this 
+ is the `{% raw %}{{email.local}}{% endraw %}` function.
```
Comment (webvictim) [inline docs/4.3/enterprise/ssh_sso.md]: Random space at the beginning of these lines

**Comment by webvictim** [comment]
There's no actual changes in this PR any more, right?

File: docs/4.3/enterprise/ssh_sso.md (hunk 1 above)
Comment (benarent) [reply docs/4.3/enterprise/ssh_sso.md]: Thanks Gus, it does look like this was merged in the great remerge of this morning.  I'm going to make a note and address this in /docs/4.3

**Pull request closed without merging**